ElectronClient/gui/NoteText.jsx

```diff
@@ -486,6 +487,10 @@ class NoteTextComponent extends React.Component {
 		return this.webviewRef_.current.wrappedInstance;
 	}
 
+	webView_focus() {
+		return this.webviewRef_.current.wrappedInstance.webviewRef_.current.focus();
```

Comment:
Please add a focus() method on the wrapped webview, and call it from here. Also are you sure that everything is always defined in this long chain?

Reply:
Sorry, laurent. But i am getting a little confused in this here. What i am understanding is the wrappedComponent is `NoteTextViewer` and you want me to add a method `focus( )` on `NoteTextViewer` and call that method from the props passed to `NoteText`. Right? Please correct me if i misunderstood at any point.

Reply:
Yes your understanding is correct.

Reply:
Thanks for telling 😄 I'll just fix this ASAP.